src/org/zaproxy/zap/extension/jython/JythonOptionParam.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2013 The ZAP development team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.jython;
+
+import org.parosproxy.paros.common.AbstractParam;
+import org.python.google.common.base.Strings;
+
+public class JythonOptionParam extends AbstractParam {
```

Comment:
- Consider renaming to `JythonOptions`;
 - It should extend VersionedAbstractParam instead. (e.g. [1])

[1] https://github.com/zaproxy/zap-extensions/blob/44f675622c0b07f6119c572f5e1fda28a88e9032/src/org/zaproxy/zap/extension/reveal/RevealParam.java#L26